src/assign.h

```diff
@@ -337,6 +337,68 @@ inline bool assign( JsonObject &jo, const std::string &name, units::mass &val,
     return true;
 }
 
+inline bool assign(JsonObject &jo, const std::string &name, units::money &val,
+    bool strict = false,
+    const units::money lo = units::money_min,
+    const units::money hi = units::money_max)
+{
+    const auto parse = [&name](JsonObject & obj, units::money & out) {
+        if (obj.has_int(name)) {
+            out = units::from_cent<std::int64_t>(obj.get_int(name));
+            return true;
+        }
+        if (obj.has_string(name)) {
+
+            out = read_from_json_string<units::money>(*obj.get_raw(name), units::money_units);
+            return true;
+        }
+        return false;
+    };
+
+    units::money out;
+
+    // Object via which to report errors which differs for proportional/relative values
+    JsonObject err = jo;
+
+    // Do not require strict parsing for relative and proportional values as rules
+    // such as +10% are well-formed independent of whether they affect base value
+    if (jo.get_object("relative").has_member(name)) {
+        units::money tmp;
+        err = jo.get_object("relative");
+        if (!parse(err, tmp)) {
+            err.throw_error("invalid relative value specified", name);
+        }
+        strict = false;
+        out = val + tmp;
+
+    }
+    else if (jo.get_object("proportional").has_member(name)) {
+        double scalar;
+        err = jo.get_object("proportional");
+        if (!err.read(name, scalar) || scalar <= 0 || scalar == 1) {
+            err.throw_error("invalid proportional scalar", name);
+        }
+        strict = false;
+        out = val * scalar;
+
+    }
+    else if (!parse(jo, out)) {
+        return false;
+    }
+
+    if (out < lo || out > hi) {
+        err.throw_error("value outside supported range", name);
+    }
+
+ /*   if (strict && out == val) {
```

Comment:
Don't comment it out, just remove it. Maybe add a @todo comment saying to put it back in.

Commented out code tends to stay forever, but becomes useless as soon as the surroundings change. Also: people won't know *why* it's there.

Reply:
Right, so instead I just set `strict` to false in the call to assign in item_factory and artifact